classes/person.rb

```diff
@@ -2,13 +2,14 @@
 
 class Person < Nameable
   attr_accessor :name, :age
-  attr_reader :id
+  attr_reader :id, :rentals
```

Comment:
- [ ] It would be better if we set the getter and setter for our rentals, since we will be reading and writing

Reply:
@addod19 Thanks for your feedback, but we are not rewriting the rentals and we can clearly push a new object to the rentals array. So it works as expected. and this way we can also avoid some unexpected scenarios like: `person.rentals = 4` which will change the data type of `rentals` from `array` to `int`.